distro/depends/lkp-dev

```diff
@@ -2,31 +2,4 @@
 # package. We can change this requiremet if we modify
 # distro/adaptation/* files to use something uniq that is not
 # part of real packages, like: |
-make
-automake
-gcc
-build-essential
-libtool
-patch
-git
-openssl
-linux-perf
-ca-certificates
-rsync
-cpio
-wget
-libklibc-dev
-linux-libc-dev
-linux-libc-dev:i386
-libc6-dev
-libc6-dev:i386
-ruby
-ruby-dev
-bzip2
-bsdtar
-fakeroot
-flex
-bison
-libssl-dev
-libipc-run-perl
-libc6-dev-x32
+
```

Comment:
thanks, why need remove these packages from lkp-dev?

Reply:
> thanks, why need remove these packages from lkp-dev?

I can see packages in lkp-dev.

$ cat distro/depends/lkp-dev 
# Put architecture packages after the non architecture specific
# package. We can change this requiremet if we modify
# distro/adaptation/* files to use something uniq that is not
# part of real packages, like: |
make
automake
gcc
build-essential
libtool
patch
git
openssl
linux-perf
ca-certificates
rsync
cpio
wget
libklibc-dev
linux-libc-dev
linux-libc-dev:i386
libc6-dev
libc6-dev:i386
ruby
ruby-dev
bzip2
bsdtar
fakeroot
flex
bison
libssl-dev
libipc-run-perl
libc6-dev-x32

Reply:
you can check commit 6acfeb1e6 not the head of pr, this means 6acfeb1e6 has problem that you can rebase it to make it clear.